### ISSUE TITLE
Unable to create the Python virtualenv

### ISSUE THREAD

**Issue by kuisathaverat** [elastic/apm-integration-testing#906]
the creation of the Python virtual environment is failing in all stages and branches, it also failed locally
https://apm-ci.elastic.co/blue/organizations/jenkins/apm-integration-tests%2Fmaster/detail/master/571/pipeline/171

```
[2020-09-02T03:21:15.684Z] Step 8/8 : USER tester
[2020-09-02T03:21:15.684Z]  ---> Running in 746996faab2d
[2020-09-02T03:21:15.684Z] Removing intermediate container 746996faab2d
[2020-09-02T03:21:15.684Z]  ---> dc060eabb7a0
[2020-09-02T03:21:15.684Z] Successfully built dc060eabb7a0
[2020-09-02T03:21:15.684Z] Successfully tagged apm-integration-testing:latest
[2020-09-02T03:21:15.684Z] mkdir -p -m 777 "/var/lib/jenkins/workspace/tegration-test-downstream_master/src/github.com/elastic/apm-integration-testing/tests/results"
[2020-09-02T03:21:15.684Z] chmod 777 "/var/lib/jenkins/workspace/tegration-test-downstream_master/src/github.com/elastic/apm-integration-testing/tests/results"
[2020-09-02T03:21:15.684Z] docker run \
[2020-09-02T03:21:15.684Z] 	--name=apm-integration-testing \
[2020-09-02T03:21:15.684Z] 	--network=apm-integration-testing \
[2020-09-02T03:21:15.684Z] 	--security-opt seccomp=unconfined \
[2020-09-02T03:21:15.684Z] 	-e APM_SERVER_URL=${APM_SERVER_URL} \
[2020-09-02T03:21:15.684Z] 	-e ES_URL=${ES_URL} \
[2020-09-02T03:21:15.684Z] 	-e KIBANA_URL=${KIBANA_URL} \
[2020-09-02T03:21:15.684Z] 	-e DJANGO_URL=http://djangoapp:8003 \
[2020-09-02T03:21:15.684Z] 	-e DOTNET_URL=http://dotnetapp:8100 \
[2020-09-02T03:21:15.684Z] 	-e EXPRESS_URL=http://expressapp:8010 \
[2020-09-02T03:21:15.684Z] 	-e FLASK_URL=http://flaskapp:8001 \
[2020-09-02T03:21:15.684Z] 	-e GO_NETHTTP_URL=http://gonethttpapp:8080 \
[2020-09-02T03:21:15.684Z] 	-e JAVA_SPRING_URL=http://javaspring:8090 \
[2020-09-02T03:21:15.684Z] 	-e RAILS_URL=http://railsapp:8020 \
[2020-09-02T03:21:15.684Z] 	-e RUM_URL=http://rum:8000 \
[2020-09-02T03:21:15.684Z] 	-e PYTHONDONTWRITEBYTECODE=1 \
[2020-09-02T03:21:15.684Z] 	-e PYTHONHTTPSVERIFY=1 \
[2020-09-02T03:21:15.684Z] 	-e ENABLE_ES_DUMP=true \
[2020-09-02T03:21:15.684Z] 	-e ES_USER=${ES_USER} \
[2020-09-02T03:21:15.684Z] 	-e ES_PASS=${ES_PASS} \
[2020-09-02T03:21:15.684Z] 	-e ELASTIC_APM_SECRET_TOKEN=${ELASTIC_APM_SECRET_TOKEN} \
[2020-09-02T03:21:15.684Z] 	-v "/var/lib/jenkins/workspace/tegration-test-downstream_master/src/github.com/elastic/apm-integration-testing/tests/results":"/app/tests/results" \
[2020-09-02T03:21:15.684Z] 	--rm \
[2020-09-02T03:21:15.684Z] 	--entrypoint make \
[2020-09-02T03:21:15.684Z] 	apm-integration-testing \
[2020-09-02T03:21:15.684Z] 	test-agent-go
[2020-09-02T03:21:15.684Z] test -d ./venv || virtualenv -q --python=python3 ./venv;\
[2020-09-02T03:21:15.684Z] source ./venv/bin/activate || exit 1;\
[2020-09-02T03:21:15.684Z] pip install -q -r requirements.txt;\
[2020-09-02T03:21:15.684Z] touch ./venv;\
[2020-09-02T03:21:15.684Z] 
[2020-09-02T03:21:15.684Z] Traceback (most recent call last):
[2020-09-02T03:21:15.684Z]   File "/usr/local/lib/python3.7/site-packages/virtualenv.py", line 24, in <module>
[2020-09-02T03:21:15.684Z]     import distutils.spawn
[2020-09-02T03:21:15.684Z]   File "/usr/local/lib/python3.7/site-packages/_distutils_hack/__init__.py", line 83, in create_module
[2020-09-02T03:21:15.684Z]     return importlib.import_module('._distutils', 'setuptools')
[2020-09-02T03:21:15.684Z]   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
[2020-09-02T03:21:15.684Z]     return _bootstrap._gcd_import(name[level:], package, level)
[2020-09-02T03:21:15.684Z] ModuleNotFoundError: No module named 'setuptools'
[2020-09-02T03:21:15.943Z] /bin/bash: line 1: ./venv/bin/activate: No such file or directory
[2020-09-02T03:21:15.943Z] make: *** [Makefile:51: venv] Error 1
[2020-09-02T03:21:16.202Z] Makefile:119: recipe for target 'dockerized-test' failed
[2020-09-02T03:21:16.202Z] make[1]: *** [dockerized-test] Error 2
[2020-09-02T03:21:16.202Z] make[1]: Leaving directory '/var/lib/jenkins/workspace/tegration-test-downstream_master/src/github.com/elastic/apm-integration-testing'
[2020-09-02T03:21:16.202Z] Makefile:116: recipe for target 'docker-test-agent-go' failed
[2020-09-02T03:21:16.202Z] make: *** [docker-test-agent-go] Error 2
```

**Comment by kuisathaverat** [comment]
I guess is this bug https://github.com/pypa/setuptools/issues/2352

**Comment by kuisathaverat** [comment]
the 50.x versions are broken so the solution is to fix the version to `setuptools==49.6.0`
